en/services/mission.md

```diff
@@ -297,10 +302,57 @@ The protocol has been implemented in C.
 Source code:
 * [src/MissionManager/PlanManager.cc](https://github.com/mavlink/qgroundcontrol/blob/master/src/MissionManager/PlanManager.cc)
 
+
 ### ArduPilot
 
+ArduPilot implements the mission protocol in C.
```

Comment:
@WickedShell, I'm hoping you can expand/confirm I captured your thoughts correctly below.

Reply:
It's C++, but I'm not sure that implementation language matters here at all. Are you trying to refer to the MAVLink binding layers? (in which case it's C) As far as docs are concerned ArduPilot could be doing OCaml with C bindings and still present the same over MAVLink, so I don't see the value of what is used to implement a sub protocol here in the docs.

Reply:
@WickedShell My logic in including this was that if someone wants to copy the code for their implementation, knowing the programming language is useful. Not a big deal in this case where there are only C++ implementations of the flight stack side, but could be if there were Python etc implementations. Happy to remove if you think this is pointless.